zipline/finance/slippage.py

```diff
@@ -502,3 +502,30 @@ def get_simulated_impact(self,
     def get_txn_volume(self, data, order):
         volume = data.current(order.asset, 'volume')
         return volume * self.volume_limit
+
+
+class FixedBasisPointsSlippage(SlippageModel):
+    """
+    Model slippage as a fixed percentage of fill price.
+
+    Parameters
+    ----------
+    bps : int, optional
+        basis points to apply
+    """
+    def __init__(self, bps=5):
```

Comment:
I would call this `basis_points` to match the `__repr__`.

Reply:
fixed